.storybook/config.js

```diff
@@ -21,20 +21,10 @@ addParameters({
 
 addDecorator(addReadme);
 
-const materials = require.context('../src/components/00-materials', true, /(story\.(js|jsx)|demo.(js|jsx))$/);
-const atoms = require.context('../src/components/10-atoms', true, /(story\.(js|jsx)|demo.(js|jsx))$/);
-const molecules = require.context('../src/components/20-molecules', true, /(story\.(js|jsx)|demo.(js|jsx))$/);
```

Comment:
Is this to remove the atoms, materials and organsims folders from the folder structure?

Reply:
No, this just loops through the folders from a higher level.

Instead of saying:
```
check everything from folder src/components/atoms
check everything from folder src/components/molecules
check everything from folder src/components/organisms
check everything from folder src/components/demos
```

I just say `check everything from src/components`

Reply:
Okay nice!